demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -120,6 +153,7 @@ export function ProductsGrid() {
             field: "additionalTypes",
             headerName: "Additional Types",
             width: 150,
+            valueFormatter: ({ value }) => value.join(", "),
             renderCell: (params) => <>{params.row.additionalTypes.join(", ")}</>,
```

Comment:
renderCell is not needed anymore

Reply:
you're right. but it's a bit annoying that our (seemingly) standard way of defining grid-cells is incompatible with the excel-export feature. So should we actually generally prefer using valueGetter and valueFormatter over renderCell? or, if we need to stick with the string columns add dedicated getExcelExportValue? Maybe even both to allow support adjusting values for excel-export...

Reply:
> you're right. but it's a bit annoying that our (seemingly) standard way of defining grid-cells is incompatible with the excel-export feature.

then we have to change out standard way

> So should we actually generally prefer using valueGetter and valueFormatter over renderCell?

we should follow https://mui.com/x/react-data-grid/faq/#what-is-the-difference-between-rendercell-valuegetter-and-valueformatter-which-one-should-i-use